client/src/index.js

```diff
@@ -1,21 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { BrowserRouter } from 'react-router-dom';
-import { Provider } from 'react-redux';
-import { createStore, applyMiddleware } from 'redux';
-import promiseMiddleware from 'redux-promise';
-import ReduxThunk from 'redux-thunk';
 
-import reducers from './reducers'; 
 import Routes from './routes';
-
-const createStoreWithMiddleware = applyMiddleware(promiseMiddleware, ReduxThunk)(createStore);
+import Root from './../src/hoc/root';
```

Comment:
Not sure if this change is needed after I have found another way how to test my code, but It is ok if I leave it like this